src/github3/users.py

```diff
@@ -742,6 +742,23 @@ def _update_attributes(self, user):
         if self.plan is not None:
             self.plan = Plan(self.plan, self)
 
+    def repository_invitations(self, number=-1, etag=None):
```

Comment:
Silly question, but should this method have the `requires_auth` decorator? I guess that if one has an `AuthenticatedUser` in their hand it's a given that they authenticated at some point, but I can see an
argument for being even more explicit about it...

Reply:
I've had hopes of using the decorator to update docstrings so that users can see in the documentation which methods require them to have valid authentication and which can be used anonymously. I think on similar objects we haven't been as disciplined so it's okay to simply ignore it here.

Reply:
Ah, so here's the other thing. Right now `GitHub` has most of the endpoints that are `/user/repository_invitations` like. So this means that this method probably belongs there and should also probably have the `@requires_auth` decorator.

Reply:
> Right now `GitHub` has most of the endpoints that are `/user/repository_invitations` like.

Ah, silly me for missing this! I was surprised to see no methods in `AuthenticatedUser`, I should have noticed this...

Reply:
No worries :)